ci/openshift-ci/bisect-range.sh

```diff
@@ -16,8 +16,14 @@ REPO="quay.io/kata-containers/kata-deploy-ci"
 TAGS=$(skopeo list-tags "docker://$REPO")
 # Only amd64
 TAGS=$(echo "$TAGS" | jq '.Tags' | jq "map(select(endswith(\"$ARCH\")))" | jq -r '.[]')
+# Sort by git
+SORTED=""
+for TAG in $(git log --merges --pretty=format:%H --reverse); do
+	[[ "$TAGS" =~ "$TAG" ]] && SORTED+="
+kata-containers-$TAG-$ARCH"
```

Comment:
Hi @ldoktor ! do you really want this line break?

Reply:
yes, because I want the `SORTED` items to be line-separated (I could use `$(echo -ne '\nkata-containers-$TAG...` for that but this looked IMO better).